Infrastructure/ApplicationDbContext.cs

```diff
@@ -32,7 +32,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<Product>()
                 .Property(b => b.Price)
-                    .HasPrecision(2,14)
+                    .HasPrecision(14,2)
```

Comment:
did you see my comment to the previous PR about chaining the calls to modelBuilder?